### PR TITLE
🐛 Use keyfile format for CentOS network config

### PR DESCRIPTION
**Pull request by lentzi90**
**What this PR does / why we need it**:

The network config format was already changed in metal3-dev-env. We need to use the same here or it will be ignored and break the tests.

Also fixes the systemctl commands for enabling the monitor.keepalived unit. It looks like there has been some changes in systemd which make `systemctl enable monitor.keepalived.service` fail because it is operating on a symlink.

```
Failed to enable unit: Refusing to operate on alias name or linked unit file: monitor.keepalived.service
```

My understanding is, that instead of first linking and then enabling the link, we should just enable the unit using the absolute path. This also creates a symlink.
This issue is not *yet* seen on Ubuntu because it has an older systemd version.

See:
- https://github.com/metal3-io/metal3-dev-env/pull/1009
- https://github.com/metal3-io/metal3-dev-env/pull/1062

### CODE REVIEW

**Comment by lentzi90** [comment]
/hold
We don't have the jinja templating in place for this to work. I need to do it in a different way.

File: scripts/ci-e2e.sh
Comment (mboukhalfa) [inline scripts/ci-e2e.sh:61]:
```suggestion
export GINKGO_FOCUS="remediation"
```

**Comment by mboukhalfa** [comment]
@lentzi90 Are you planning to parametrize the hardcoded config on this PR or merge that to fix CI that follow up with paramterization PR ? 
+1 for unlocking the CI now

**Review by furkatgofurov7 — APPROVED** [review]
Thank you. Let's make sure to backport this to the `release-1.1` branch once merged to get the CI back there too. 

/approve

**Comment by furkatgofurov7** [comment]
> Are you planning to parametrize the hardcoded config on this PR or merge that to fix CI than follow up with paramterization PR ?

@mboukhalfa which config are you referring to?

**Comment by lentzi90** [comment]
Let's merge it now and discuss the parameterization separately
/hold cancel

**Pull request merged**
